controller/controller_server/utils.py

```diff
@@ -154,14 +153,48 @@ def validate_delete_snapshot_request(request):
     logger.debug("request validation finished.")
 
 
+def validate_validate_volume_capabilities_request(request):
+    logger.debug("validating validate_volume_capabilities volume request")
+
+    logger.debug("validating volume id")
+    volume_id = request.volume_id
+    if not request.volume_id:
+        raise ValidationException(messages.name_should_not_be_empty_message)
+    if config.PARAMETERS_OBJECT_ID_DELIMITER not in volume_id:
+        raise ObjectIdError(config.OBJECT_TYPE_ID_VOLUME, volume_id)
+
+    logger.debug("validating volume capabilities")
+    if not request.volume_capabilities:
```

Comment:
consider moving this inside `validate_csi_volume_capabilties`

Reply:
maybe renaming to `validate_csi_volume_capabilities` too

Reply:
done